python/tk_multi_about/dialog.py

```diff
@@ -104,7 +104,7 @@ def show_in_fs(self):
         paths = self._app.context.filesystem_locations
         for disk_location in paths:
             if not QtGui.QDesktopServices.openUrl(
-                "file://{0}".format(disk_location).replace("\\", "/")
+                "file:///{0}".format(disk_location).replace("\\", "/")
```

Comment:
I'm not an expert on this file URI format, but according to here:
https://en.wikipedia.org/wiki/File_URI_scheme
The following are valid:

> file:/path, file:///path or file://hostname/path`

What concerns me a bit about this change is, what if the disk_location starts with a `/`?
You then end up with four does that still work?
What is the issue this is trying to fix?

Reply:
I just tried with my advanced setup on Windows who start with \\[machine name]\...\
and it fails with this error: // Error: Shotgun tk-multi-about: Failed to open '\\remmj06fbt9\G\ShotgunNiruhtam\mnt\projects\testcreate'! //

Reply:
@pscadding It is currently broken on Windows with letter drive mapping.
@mathurf thanks for that, I will test some more cases with UNC, etc.